primitives/src/sybil.rs

```diff
@@ -1,34 +1,100 @@
 use codec::{Decode, Encode};
+use fixed::traits::Fixed;
 use rstd::vec::Vec;
-use sp_core::RuntimeDebug;
+use sp_core::{RuntimeDebug, H256};
 
 use crate::ceremonies::ProofOfAttendance;
-use crate::communities::CommunityIdentifier;
 use crate::scheduler::CeremonyIndexType;
 
-pub type ProofOfPersonhoodRequest<Signature, AccountId> =
-    Vec<(CommunityIdentifier, ProofOfAttendance<Signature, AccountId>)>;
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Default, RuntimeDebug)]
+pub struct IssueProofOfPersonhoodConfidenceCall<Signature, AccountId> {
+    call_index: [u8; 2],
+    requester: AccountId,
```

Comment:
is this the accountid the subject uses on the faucet parachain? why is that needed?
encointer advises users to use a different accountid for each ceremony for improved privacy, so all proofs can have different accountid. And we would also advise to use yet another accountid on the faucet parachain.
The question is: why would the encointer chain need to know this accountid at all?

Reply:
Yes, this is absolutely unnecessary, this has only been used as the key for the `PendingRequests`, which is a bad choice.

This could hash of the ProofOfPersonHood request, which makes more sense.